docs/content/docs/3.composables/use-scroll-shadow.md

```
@@ -0,0 +1,112 @@
+### Parameters
+
+::field-group
+
+  ::field{name="element" type="MaybeRef<HTMLElement | null | undefined>" required}
+  A template ref or reactive reference to the scrollable element.
+  ::
+
+  ::field{name="options" type="UseScrollShadowOptions"}
+  Configuration options for the scroll shadow.
+
+    ::collapsible
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)